### PR TITLE
PWGHF: Adding centrality in the B+ task

### PR DESCRIPTION
**Pull request by deepathoms**
Include centrality to the task and add 3rd axis to hMass histogram to include centrality

### CODE REVIEW

File: PWGHF/Tasks/taskBPlus.cxx [hunk 1]
```diff
@@ -72,8 +75,11 @@ struct HfTaskBplus {
 
   Filter filterSelectCandidates = (aod::hf_selcandidate_bplus::isSelBPlusToD0Pi >= selectionFlagBPlus);
 
-  void process(soa::Filtered<soa::Join<aod::HfCandBPlus, aod::HFSelBPlusToD0PiCandidate>> const& candidates, soa::Join<aod::HfCandProng2, aod::HFSelD0Candidate>, aod::BigTracks)
+  void process(soa::Join<aod::Collisions, aod::CentV0Ms>::iterator const& collision, soa::Filtered<soa::Join<aod::HfCandBPlus, aod::HFSelBPlusToD0PiCandidate>> const& candidates, soa::Join<aod::HfCandProng2, aod::HFSelD0Candidate>, aod::BigTracks)
   {
+    float centrality = collision.centV0M();
```
Comment (vkucera) [inline PWGHF/Tasks/taskBPlus.cxx:80]:
```suggestion
    auto centrality = collision.centV0M();
```
Better to keep the type automatically deduced.

**Review by vkucera — CHANGES_REQUESTED** [review]

File: PWGHF/TableProducer/HFCandidateCreatorBPlus.cxx [hunk 2]
```diff
@@ -114,8 +114,8 @@ struct HfCandidateCreatorBplus {
       auto prong1TrackParCov = getTrackParCov(prong1);
       auto collision = prong0.collision();
 
-      // LOGF(info, "All track: %d (prong0); %d (prong1)", candidate.index0().globalIndex(), candidate.index1().globalIndex());
-      // LOGF(info, "All track pT: %f (prong0); %f (prong1)", prong0.pt(), prong1.pt());
+      // LOGF(INFO, "All track: %d (prong0); %d (prong1)", candidate.index0().globalIndex(), candidate.index1().globalIndex());
+      // LOGF(INFO, "All track pT: %f (prong0); %f (prong1)", prong0.pt(), prong1.pt());
```
Comment (vkucera) [inline PWGHF/TableProducer/HFCandidateCreatorBPlus.cxx:118]: Why did you change these? See https://github.com/AliceO2Group/O2Physics/pull/102/files

**Review by vkucera — CHANGES_REQUESTED** [review]

**Review by fcolamar — COMMENTED** [review]
Hi Deepa, your code updates are ok to me, pending the comment by Vit.
Maybe just cross-check with the experts whether for ALICE3 the usage of centrality is now ok (there was some issue with centV0M for some MC samples in the past) or it's safer to switch to multiplicity.
There are several style updates for commented lines, this happened also to a recent commit of mine, maybe something has changed with cling behaviour, anyway I don't see it as an issue

**Comment by deepathoms** [comment]
Hi @vkucera @fcolamar. I have fixed the deprecated uppercase issue. I also checked with David about the centrality estimation using V0M. He said it is ok, but I should check it again by running on a MC sample and follow up with him. Thank you
